app/views/api_docs/vendor_api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,12 @@
+## 22nd March
+
+The following experimental/sandbox endpoint has been updated:
+
+- `/test-data/generate` is now asynchronous. A POST request to this endpoint will queue a job to generate the specified test applications.
+The list of created application IDs will no longer be returned.
+Any applications will become available as soon as they have been generated.
+Applications generated in this way will now have their `updated_at` set to the current time, so they can be retrieved using the `GET /applications` endpoint with the `since` parameter.
```

Comment:
Maybe bullet these? Markdown collapses them all onto one line, which is less readable